src/main/java/roomescape/payment/PaymentErrorHandler.java

```diff
@@ -0,0 +1,45 @@
+package roomescape.payment;
+
+
+import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import lombok.Getter;
+import org.springframework.http.client.ClientHttpResponse;
+import org.springframework.web.client.DefaultResponseErrorHandler;
+import roomescape.common.exception.PaymentException;
+
+public class PaymentErrorHandler extends DefaultResponseErrorHandler {
+
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    @Override
+    public void handleError(final ClientHttpResponse response) throws IOException {
+        if (response.getStatusCode().is2xxSuccessful()) {
```

Comment:
```suggestion
        if (!response.getStatusCode().isError()) {
```
4XX, 5XX만 걸러내려면 이게 더 명확할 수도 있겠네요!

Reply:
생각해보니 기존 코드는 3xx번대 응답도 에러로 처리하고 있겠네요..
수정했습니다!